README.md

```diff
@@ -4,7 +4,7 @@ aprilaire-ha is a custom integration for [Home Assistant](https://www.home-assis
 
 # Compatibility
 
-aprilaire-ha is compatible with many models of Aprilaire thermostats. It is tested with an Aprilaire 6045m, but it should work with other 6000- and 8000-series thermostats.
+aprilaire-ha is compatible with Aprilaire [Home Automation](https://www.aprilairepartners.com/technical-information-options/home-automation-technical-information) 8800 series and [Zone Control](https://www.aprilairepartners.com/technical-information-options/zoning-technical-information) 6000 series thermostats.
```

Comment:
@nberardi I would like to keep which devices it has been tested with. While it _should_ be compatible with the whole series, I/we haven't tested on all of the models, so I'd like to avoid implications that it has. Perhaps a section below for "Tested Models"?

Reply:
I think that is a good addition to mention verified devices.